README.md

```diff
@@ -82,22 +82,58 @@ Navigate to http://localhost:8080/opsui, you should see the default OPSUI system
 
 To contribute a patch, follow these instructions (note that installing
 [Hub](http://hub.github.com) is not strictly required, but is recommended).
+```
+Download and install hub.github.com
+```
+1.File JIRA issue for your fix at https://issues.apache.org/jira/browse/OODT.
```

Comment:
@NGimhana Could you please add a space after the number (please do the same for all the numbers.)
```suggestion
1. File JIRA issue for your fix at https://issues.apache.org/jira/browse/OODT.
```

Reply:
Fixed as suggested.